src/api/setDoc.ts

```diff
@@ -21,17 +21,20 @@ export async function saveUserToDb(
     const userData: User = {
       firstName,
       lastName,
-      phoneNum,
       login,
       email: user.email || '',
       id: user.uid,
       role,
-      avatarUrl,
     };
+    if (phoneNum !== undefined) {
+      userData.phoneNum = phoneNum;
+    }
+    if (avatarUrl !== undefined) {
+      userData.avatarUrl = avatarUrl;
+    }
```

Comment:
```suggestion
    if (!phoneNum) {
      userData.phoneNum = phoneNum;
    }
    if (!avatarUrl) {
      userData.avatarUrl = avatarUrl;
    }
```

Reply:
when !phoneNum can't register user in db